source/_components/mychevy.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: page
+title: "MyChevy"
+description: "Instructions on how to integrate Chevy Bolt car into Home Assistant."
+date: 2017-08-28 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tesla.png
+ha_category: Hub
+ha_release: 0.61
+ha_iot_class: "Cloud Polling"
+---
+
+The `MyChevy` component communicates with the
+[my.chevrolet](https://my.chevrolet.com) website using selenium to log
+in as a your user, and screen scrape the data provided. GM does not
+make it easy to sign up for any official development program, so this
+provides a work around to get access to your data.
+
+This component provides the following platforms:
+ - Binary sensors - if the car is plugged in
+ - Sensors - such as Battery Level, Charge Mode, EST Range, Total
+   Distance Traveled
+
+To use MyChevy in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+mychevy:
+  username: email
+  password: password
+```
+
+Configuration variables:
```

Comment:
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

Reply:
ack

Reply:
@sdague You still need to remove this header...